src/components/Modeler.vue

```diff
@@ -336,10 +336,20 @@ export default {
       const registeredIndex = node.inspectorConfig[0].items.findIndex(item => {
         return config.id && config.id === item.id;
       });
+        
+      let items;
+      if (config.container) {
```

Comment:
There's two if/else blocks in this function and it may help readability to break these out into separate functions.

I've recently learned about the idea of command query separation; having separate functions for executing an action or retrieving data. And it seems the 1st if/else block can qualify as a query and the 2nd as a command.